racer_types/Animal.py

```diff
@@ -6,3 +6,5 @@ class Animal(NamedTuple):
     steps_per_interval: int
     interval_spacing: float
     starting_position: int
+    passed_out_time: float
```

Comment:
all animals have `passed_out_time` even if they won't pass ever pass out. If he thought of 10 other optional attributes, would all animals need those too?